lib/main.dart

```diff
@@ -64,6 +65,11 @@ class _AppState extends ConsumerState<App> with WidgetsBindingObserver {
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setPreferredOrientations(
+      [
+        DeviceOrientation.portraitUp
```

Comment:
Does this lock the device to portrait mode?

Reply:
Yes. I could make that only apply to mobile platforms if you want.

Reply:
it seems great how it is